apps/auth/oauth2.py

```
@@ -0,0 +1,27 @@
+from authlib.integrations.starlette_client import OAuth, OAuthError
```

Ruff (0.8.2)

`authlib.integrations.starlette_client.OAuthError` imported but unused

Remove unused import: `authlib.integrations.starlette_client.OAuthError`

(F401)